### PR TITLE
Fix .webapp-nextcloud-<version> integrity error

### PR DESCRIPTION
**Pull request by golodhrim**
On Gentoo/Funtoo Systems there is next to a .webapp file also a .webapp-nextcloud-<version> file. upgraded from Owncloud they had the following fix in it. hope it can be included

### CODE REVIEW

File: lib/private/IntegrityCheck/Iterator/ExcludeFileByNameFilterIterator.php [hunk 1]
```diff
@@ -43,6 +43,17 @@ class ExcludeFileByNameFilterIterator extends \RecursiveFilterIterator {
 		'.directory', // Dolphin (KDE)
 		'.webapp', // Gentoo/Funtoo & derivatives use a tool known as webapp-config to manager wep-apps.
 	];
+	/**
+	 * Array of excluded file names. Those are not scanned by the integrity checker.
+         * This is used to exclude files which administrators could upload by mistakes
+         ￼* such as .DS_Store files. These strings are submatched, so any file names
+￼         * containing these strings, are ignored.
```
Comment (nickvergessen) [inline lib/private/IntegrityCheck/Iterator/ExcludeFileByNameFilterIterator.php]: weird characters on these lines, please delete

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Comment by LukasReschke** [comment]
I'm absolutely not a fan of this, can distributors please stop putting random files into the source code?!?

**Comment by MorrisJobke** [comment]
> I'm absolutely not a fan of this, can distributors please stop putting random files into the source code? But whatever … 😢

Is this good for review now?

**Comment by LukasReschke** [comment]
This is a duplicate of https://github.com/nextcloud/server/pull/3113 and also opens a security hole.

**Pull request closed without merging**